tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_default_parameters.bal

```diff
@@ -270,6 +270,36 @@ public function sleep(int millis) = @java:Method {
     'class: "org.ballerinalang.test.utils.interop.Utils"
 } external;
 
+function testFuncWithSpecificFieldInRecordLiteralForDefaultValue() {
```

Comment:
Specific field is both key-value pair and variable name field. I would use VariableNameField instead.

https://ballerina.io/spec/lang/master/#specific-field